tests/jvm/src/test/scala/io/iteratee/files/TryTests.scala

```diff
@@ -0,0 +1,26 @@
+package io.iteratee.files
+
+import io.iteratee.{ Iteratee, Enumerator }
+import io.iteratee.tests.BaseSuite
+import scala.util.{ Success, Try }
+import java.io.File
+
+class TryTests extends BaseSuite {
+  val tryModule = FileModule[Try]
```

Comment:
would be interesting to benchmark this. My guess is a strict monad (like Option, Either or Try) could be faster for single threaded applications.